test/sign-and-verify-tests.c

```diff
@@ -53,12 +53,14 @@ typedef struct sign_and_verify_fixture {
 static void setup(sign_and_verify_fixture* fixture);
 static void teardown(sign_and_verify_fixture* fixture);
 
-int main()
+int main(int argc, char* argv[])
 {
     sign_then_verify_good();
     sign_then_verify_on_rev_list();
-    sign_benchmark();
-    verify_benchmark();
+    if (argc > 1 && 0 == strcmp(argv[1], "benchmark")) {
```

Comment:
Maybe split the benchmarks into a separate file and exclude that from the `ctest -T memcheck` using the `-E` flag?

Reply:
Good call.